<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtPropGen</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,10,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Property name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lName">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Property type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lType">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cWrite">
         <property name="text">
          <string>WRITE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cChain">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Chain</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cReset">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cNotify">
       <property name="text">
        <string>NOTIFY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cEmitType"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cRev">
         <property name="text">
          <string>REV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sRev">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cConstant">
       <property name="text">
        <string>CONSTANT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cUser">
       <property name="text">
        <string>USER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cFinal">
       <property name="text">
        <string>FINAL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cDesignable">
       <property name="text">
        <string>DESIGNABLE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cStored">
       <property name="text">
        <string>STORED</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cScriptable">
       <property name="text">
        <string>SCRIPTABLE</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cComment">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="pComment">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pDelete">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lName</tabstop>
  <tabstop>lType</tabstop>
  <tabstop>cWrite</tabstop>
  <tabstop>cChain</tabstop>
  <tabstop>cReset</tabstop>
  <tabstop>cNotify</tabstop>
  <tabstop>cEmitType</tabstop>
  <tabstop>cRev</tabstop>
  <tabstop>sRev</tabstop>
  <tabstop>cConstant</tabstop>
  <tabstop>cUser</tabstop>
  <tabstop>cFinal</tabstop>
  <tabstop>cDesignable</tabstop>
  <tabstop>cStored</tabstop>
  <tabstop>cScriptable</tabstop>
  <tabstop>cComment</tabstop>
  <tabstop>pComment</tabstop>
  <tabstop>pDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
